python/cudf_polars/cudf_polars/dsl/expressions/string.py

```diff
@@ -155,7 +155,7 @@ def _validate_input(self) -> None:
             target = self.children[1]
             # Above, we raise NotImplementedError if the target is not a Literal,
             # so we can safely access .value here.
-            if target.value == pa.scalar("", type=target.value.type):  # type: ignore[attr-defined]
+            if target.value == "":  # type: ignore[attr-defined]
```

Comment:
I don't know if it was related, can we remove the type annotation now?

Reply:
Apparently not. I guess mypy cannot deduce that self.children is `Collection[Literal]` after the `if not all(isinstance(expr, Literal) for expr in self.children[1:])` call